README.md

```diff
@@ -1,13 +1,11 @@
-# Project Name
+# Project Chatbot
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I wanted to do a chatbot that gives the user generated affirmations and be a guide in how to use affirmations. I originally aimed to also offer the user to calculate their personal life path number (by adding their birth day digits until its a single digit). And after that link to where they could read more about what each number means according to the ancient mythology of numerology. These funcions id like to develop more as i get to learn more Javascript and get more familliar how to handle the event listeners.
```

Comment:
I really like your ideas here for your chatbot!

Would you use a Date and Time picker and get the chatbot to read the date and calculate the number itself? Or would you get the user to input the numbers manually into a function that adds the numbers together?

Reply:
Thank you!! I thought id add a date-picker, but im not sure how to make the input into a function that calculates the numbers.. I guess ill have to first make the input into numbers and go from there, but this part im thinking im gunna have to save for later as i get a better understanding for functions. I found codes for doing the calculations, but i didnt understand it so i didnt dare to use it just yet 😜

Reply:
I understand! Look forward to seeing how it will be in the future!